dockerfiles/dali/daliconf.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The plan would be that Dali should not need a config file by default, and that all components should look more like the roxie case. I think dali needs one at present because of the need to disable whitelist

Reply:
daserver is dali